support/docker/config/production.yaml

```diff
@@ -0,0 +1,58 @@
+listen:
+  port: 9000
+
+# Correspond to your reverse proxy "listen" configuration
+webserver:
+  https: true
+  hostname: undefined
+  port: 443
+
+# Your database name will be "peertube"+database.suffix
+database:
+  hostname: 'db'
+  port: 5432
+  suffix: ''
+  username: 'postgres'
+  password: 'postgres'
+
+# From the project root directory
+storage:
+  avatars: '../data/avatars/'
```

Comment:
So this is `/usr/src/data`? You need to adjust the `VOLUME` directive in the Dockerfile accordingly. (Yes, I made a similar mistake in my PR ;))

Reply:
This is relative to ``/app``, so it is actually ``/data/avatars``.

Reply:
Ah, sorry for the noise.